src/dataManager.h

```diff
@@ -37,22 +38,45 @@ struct ControlsJson {
 		p2(p2) {}
 };
 
+struct PickupData {
+	std::string name;
+	int frame;
+	float duration;
+
+	PickupData() {}
+	PickupData(std::string n, int f, float d) : name(n), frame(f), duration(d) {}
+};
+
+struct PickupJson {
```

Comment:
Yes your point would make sense but you already have another struct called `PickupData`. Ff you add new objects to `pickup.json` that's for `PickupData` to manage. No need for any changes to be made to this struct. You can keep the name `PickupJson` if you want neatness but adding a struct here just seems to overcomplicate things.